yarn-project/boxes/blank-react/src/scripts/deploy_contract.ts

```diff
@@ -12,7 +10,9 @@ export async function deployContract(
   const tx = new DeployMethod(activeWallet.publicKey, pxe, contractArtifact, typedArgs).send({
     contractAddressSalt: salt,
   });
+  console.log('tx', tx);
   await tx.wait();
+  console.log('tx receipt', tx.getReceipt());
```

Comment:
Are these logs left here on purpose?

Reply:
oops no removing, that was debugging against published sandbox